### ISSUE TITLE
Liveness analysis ignores functions with built-in macro calls

### ISSUE THREAD

**Issue by ortem** [intellij-rust/intellij-rust#7282]
## Environment

* **IntelliJ Rust plugin version:** 0.4.148.3892-211-nightly
* **Rust toolchain version:** 1.53.0-nightly (b0c818c5e 2021-04-16) x86_64-apple-darwin
* **IDE name and version:** CLion 2021.1.1 (CL-211.7142.21)
* **Operating system:** macOS 11.2.3
* **Macro expansion engine:** new
* **Name resolution engine:** new

## Problem description
Liveness analysis does not analyze functions containing any built-in (with `#[rustc_builtin_macro]` attribute) macro calls. Presumably, this happens only after #6876 (plugin version 0.4.143).

## Steps to reproduce
```Rust
fn test1() {
    let x = 42; // expected: "Variable `x` is never used", actual: no warning
    concat!("hello", 10);
}

fn test2() {
    let x = 42; // expected: "Variable `x` is never used", actual: no warning
    if cfg!(windows) {}
}
```

**Comment by ortem** [comment]
Fixed by #7288
